src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -129,62 +143,81 @@ public boolean isLoopOf(String s) {
 	}
 
 	@Override
-	@Nullable
 	@SuppressWarnings("null")
-	protected Entity[] get(Event e) {
-		if (isUsingRadius) {
-			Iterator<? extends Entity> iter = iterator(e);
+	protected Entity @Nullable [] get(Event event) {
+		if (isUsingRadius || isUsingCuboid) {
+			Iterator<? extends Entity> iter = iterator(event);
 			if (iter == null || !iter.hasNext())
 				return null;
 
-			List<Entity> l = new ArrayList<>();
+			List<Entity> list = new ArrayList<>();
 			while (iter.hasNext())
-				l.add(iter.next());
-			return l.toArray((Entity[]) Array.newInstance(returnType, l.size()));
+				list.add(iter.next());
+			return list.toArray((Entity[]) Array.newInstance(returnType, list.size()));
 		} else {
 			if (chunks != null) {
-				return EntityData.getAll(types.getArray(e), returnType, chunks.getArray(e));
+				return EntityData.getAll(types.getArray(event), returnType, chunks.getArray(event));
 			} else {
-				return EntityData.getAll(types.getAll(e), returnType, worlds != null ? worlds.getArray(e) : null);
+				return EntityData.getAll(types.getAll(event), returnType, worlds != null ? worlds.getArray(event) : null);
 			}
 		}
 	}
 
 	@Override
 	@Nullable
 	@SuppressWarnings("null")
-	public Iterator<? extends Entity> iterator(Event e) {
+	public Iterator<? extends Entity> iterator(Event event) {
 		if (isUsingRadius) {
 			assert center != null;
-			Location l = center.getSingle(e);
-			if (l == null)
+			Location location = center.getSingle(event);
+			if (location == null)
 				return null;
 			assert radius != null;
-			Number n = radius.getSingle(e);
-			if (n == null)
+			Number number = radius.getSingle(event);
+			if (number == null)
 				return null;
-			double d = n.doubleValue();
+			double rad = number.doubleValue();
 
-			if (l.getWorld() == null) // safety
+			if (location.getWorld() == null) // safety
 				return null;
 
-			Collection<Entity> es = l.getWorld().getNearbyEntities(l, d, d, d);
-			double radiusSquared = d * d * Skript.EPSILON_MULT;
-			EntityData<?>[] ts = types.getAll(e);
-			return new CheckedIterator<>(es.iterator(), e1 -> {
-					if (e1 == null || e1.getLocation().distanceSquared(l) > radiusSquared)
+			Collection<Entity> nearbyEntities = location.getWorld().getNearbyEntities(location, rad, rad, rad);
+			double radiusSquared = rad * rad * Skript.EPSILON_MULT;
+			EntityData<?>[] entityTypes = types.getAll(event);
+			return new CheckedIterator<>(nearbyEntities.iterator(), entity -> {
+					if (entity == null || entity.getLocation().distanceSquared(location) > radiusSquared)
 						return false;
-					for (EntityData<?> t : ts) {
-						if (t.isInstance(e1))
+					for (EntityData<?> entityType : entityTypes) {
+						if (entityType.isInstance(entity))
 							return true;
 					}
 					return false;
 				});
+		} else if (isUsingCuboid) {
+			assert from != null;
```

Comment:
These assertions aren't necessarily a good thing to be doing in a main execution point where it'll only be caught by 1 test. It's just extra.